spec/index.html

```diff
@@ -392,7 +392,12 @@ <h3>Document Conventions</h3>
         <h3>Terminology</h3>
         <dl>
           <dt>SPARQL Protocol client</dt>
-          <dd>An HTTP client (as defined by <a data-cite="rfc9110#section-3.3">RFC 9110</a>) that sends HTTP requests for SPARQL
+          <dd>An HTTP client (as defined by
+            <a data-cite="rfc9110#section-3.3">Section 3.3. Connections, Clients, and
+            Servers</a> of [[[RFC9110]]] [[RFC9110]]) that sends HTTP requests for
+            SPARQL Protocol operations. (Also known as: <em>client</em>.)</dd>
```

Comment:
This seems to duplicate the next paragraph

Reply:
Yes. I had made a [suggestion](https://github.com/w3c/sparql-protocol/pull/22/files#r1333385992), which was meant to lead to a decision between the paragraph above and the paragraph below, which are better compared [in the preview](https://pr-preview.s3.amazonaws.com/w3c/sparql-protocol/pull/22.html#terminology). I prefer the top/first paragraph (which includes the title of the cited section, as well as the title of the RFC, along with the RFC####). If this is accepted/adopted, then I think the other citations should be tweaked in similar fashion.